SpatialGDK/Source/SpatialGDK/Public/Utils/RepLayoutUtils.h

```diff
@@ -192,7 +192,14 @@ inline TArray<UFunction*> GetClassRPCFunctions(const UClass* Class)
 	}
 
 	// When using multiple EventGraphs in blueprints, the functions could be iterated in different order, so just sort them alphabetically.
-	RelevantClassFunctions.Sort([](const UFunction& A, const UFunction& B) { return A.GetFName() < B.GetFName(); });
+	RelevantClassFunctions.Sort([](const UFunction& A, const UFunction& B)
+	{
+#if ENGINE_MINOR_VERSION <= 22
+		return A.GetFName() < B.GetFName();
+#else
+		return FNameLexicalLess()(A.GetFName(), B.GetFName());
```

Comment:
Was the FName < operator removed in 4.23?

Reply:
It was deprecated, since it does expensive lexical sorting, but that is exactly the functionality we want. There is also FNameFastLess() which is faster, but is not stable between runs (so not usable for us).